{
  "path": "/documentation/xctest/xctest/3521151-teardownwitherror",
  "type": "Instance Method",
  "name": "tearDownWithError()",
  "desc": "Provides an opportunity to perform cleanup and to throw errors after each test method in a test case ends.",
  "items": [],
  "declaration": "func tearDownWithError() throws"
}